hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -747,6 +747,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+en_scb=0"]
     }
+    {
+      name: chip_sw_alert_handler_entropy
+      uvm_test_seq: chip_sw_alert_handler_entropy_vseq
+      sw_images: ["//sw/device/tests/sim_dv:alert_handler_entropy_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+en_scb=0"]
```

Comment:
Could you explain why you disable the scoreboard?

Reply:
Sure! It is mainly because we cannot monitor the cross domain alerts really accurately from the scb side. So it might lead to some false DV testbench error.